cypress/page/login.page.ts

```diff
@@ -0,0 +1,23 @@
+class LoginPage{
+    private email: string;
+    private passwd: string;
+    private examplemail: string;
+    private examplepasswd: string;
+    private submitButton: string;
+
+    constructor(){
+        this.email = "#email";
+        this.passwd = "#passwd";
+        this.examplemail = "aperdomobo@gmail.com";
```

Comment:
This isn't really part of the page, it's data for the test, so that's why I'd recommend you receive these params in the `signIn()` function